default.nix

```diff
@@ -0,0 +1,5 @@
+# nix-shell --pure --command 'cd src; make'
+let
+  nixpkgs = import <nixpkgs> {};
+in
+  nixpkgs.callPackage ./hello.nix {}
```

Comment:
Is this supposed to be calling `hello.nix`?

Reply:
No. I changed now.